launch/model_endpoint.py

```diff
@@ -83,8 +101,14 @@ def predict(self, request: EndpointRequest) -> EndpointResponse:
         )
 
     def status(self):
-        # TODO this functionality doesn't exist serverside
-        raise NotImplementedError
+        """Gets the status of the ModelEndpoint.
+        
+        Returns:
+            An object containing the ModelEndpoint name, the currently deployed bundle, 
+            and the status of the creation task.
+        """
+        resp = self.connection.get(f"{ENDPOINT_PATH}/{self.endpoint_id}")
```

Comment:
1. `Sync/AsyncModel` does not have an attribute `connection`. Are you merging all your local changes?
2. My PR (needs a review 🙏 ) partially refactors `Sync/Async model` with a base class https://github.com/scaleapi/launch-python-client/pull/5/files#diff-314694e38542dd421a72c9b17390316546c94c737a5bc249e21d9a30485687a0R82 so that some of the functions like "status" can share the code between classes

Reply:
maybe you mean `self.client.connection` as opposed to `self.connection`? although I'm not sure if Ivan's changes change this